platform/mesa/cmds/osdemo_fb/osdemo_fb.c

```diff
@@ -63,38 +64,19 @@ void init_buffers(void) {
 	printf("The framebuffer device was mapped to memory successfully.\n");
 
 	sw_base = malloc(Width * Height * mesa_fbi->var.bits_per_pixel / 8);
-	fb_overlay_init(mesa_fbi, sw_base);
-}
-
-static void swap_buffers() {
-	static struct timespec time_prev;
-	struct timespec time;
-	static int fps = 0;
-	static int counter = 0;
-
-	static char fps_str[] = "FPS=??";
-
 	if (animated_scene) {
-		/* Refresh FPS counter every second */
-		clock_gettime(CLOCK_REALTIME, &time);
-
-		counter++;
-
-		if (time.tv_sec != time_prev.tv_sec) {
-			time_prev = time;
-			fps = counter > 99 ? 99 : counter;
-			counter = 0;
-		}
-
-		fps_str[4] = '0' + fps / 10;
-		fps_str[5] = '0' + fps % 10;
-		fb_overlay_put_string(0, 0, "Embox MESA demo v0.2");
-		fb_overlay_put_string(0, 1, fps_str);
+		fps_set_format("Embox MESA demo v0.2\nFPS=%2d");
```

Comment:
I think an argument for `%2d` is expected?

Reply:
Fixed this anyway :)